pkg/cmd/env/connect.go

```diff
@@ -0,0 +1,117 @@
+package env
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	"github.com/coreeng/corectl/pkg/cmdutil/userio"
+	corectlenv "github.com/coreeng/corectl/pkg/env"
+	"github.com/coreeng/corectl/pkg/gcp"
+	"github.com/coreeng/developer-platform/pkg/environment"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var localPort int
+
+func connectCmd(cfg *config.Config) *cobra.Command {
+	connectCmd := &cobra.Command{
+		Use:   "connect [environment]",
+		Short: "Connect to an environment",
+		Long:  `This command allows you to connect to a specified environment.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			io := userio.NewIOStreams(
+				cmd.InOrStdin(),
+				cmd.OutOrStdout(),
+			)
+			return connect(cfg, io)
+		},
+	}
+
+	connectCmd.Flags().IntVar(&localPort, "localPort", 54808, "The local port to use for connection to the cluster")
+
+	return connectCmd
+}
+
+func connect(cfg *config.Config, stream userio.IOStreams) error {
+	envs, err := environment.List(environment.DirFromCPlatformRepoPath(cfg.Repositories.CPlatform.Value))
+	if err != nil {
+		return err
+	}
+
+	inputEnv := createEnvInputSwitch(envs)
```

Comment:
the purpose if this input switch is to either take value from user input or from some other source (config or cmd flag).
Let's create a flag to pick required env. This way `corectl` could be used in non-interactive environments